index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
```

Comment:
Check the link on this one. It looks like you changed directory structure but did not update the link.